Example/StripeConnectExample/StripeConnectExample/Appearance/AppSettings+AppearanceInfo.swift

```diff
@@ -107,13 +107,158 @@ extension AppearanceInfo {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xF7F7F7)
+            : .init(hex: 0x6e7781)
+        }
+        appearance.colors.formAccent = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xEbF0F4)
+            : .init(hex: 0x0969DA)
+        }
+        appearance.colors.formHighlightBorder = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x363636)
+            : .init(hex: 0x0969DA)
+        }
+        appearance.colors.danger = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xDF1B41)
+            : .init(hex: 0xb35900)
+        }
+        appearance.badgeNeutral.colorBorder = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x7D7F87)
+            : .init(hex: 0x8c959f)
+        }
+        appearance.badgeNeutral.colorText = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xD1D3DC)
+            : .init(hex: 0x6e7781)
+        }
+        appearance.badgeSuccess.colorBorder = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x7A8C7B)
+            : .init(hex: 0x218bff)
+        }
+        appearance.badgeSuccess.colorText = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xCFE3D0)
+            : .init(hex: 0x0969DA)
+        }
+        appearance.badgeWarning.colorBorder = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x794A00)
+            : .init(hex: 0xd4a72c)
+        }
+        appearance.badgeWarning.colorText = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xE7A288)
+            : .init(hex: 0xbf8700)
+        }
+        appearance.badgeDanger.colorBorder = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x6F2341)
+            : .init(hex: 0xdd7815)
+        }
+        appearance.badgeDanger.colorText = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0xEC93AF)
+            : .init(hex: 0xb35900)
+        }
+        appearance.colors.offsetBackground = .init {
+            $0.userInterfaceStyle == .dark
+            ? .init(hex: 0x171717)
+            : .init(hex: 0xF8F8F8)
+        }
+
+        return .init(displayName: "Dynamic colors", appearance: appearance)
+    }
+
     static var customFont: AppearanceInfo {
         var appearance = EmbeddedComponentManager.Appearance()
         appearance.typography.font = UIFont(name: "Handjet-Regular", size: UIFont.systemFontSize)
         return .init(displayName: "Custom Font", appearance: appearance)
     }
 }
+
+extension UIColor {
+    convenience init(hex: UInt) {
+        let r = hex >> 16 & 0xFF
+        let g = hex >> 8 & 0xFF
+        let b = hex & 0xFF
+
+        self.init(red: CGFloat(r) / 255,
+                  green: CGFloat(g) / 255,
+                  blue: CGFloat(b) / 255,
+                  alpha: 1)
+    }
+}
```

Comment:
Made it easier to copy hex colors from our [theming playground](https://docs.stripe.com/connect/customize-connect-embedded-components?enable_embedded_theming_demo=true&enable_new_theming=true&enable_uiconfig_copy_link=true&initOptionsOverride=%7B%22appearance%22%3A%7B%22colorBackground%22%3A%22%23272626%22%2C%22colorPrimary%22%3A%22%23EBF0F4%22%2C%22colorBorder%22%3A%22%233D3D3D%22%2C%22badgeSuccessTextColor%22%3A%22%23CFE3D0%22%2C%22badgeNeutralTextColor%22%3A%22%23D1D3DC%22%2C%22badgeWarningTextColor%22%3A%22%23E7A288%22%2C%22badgeDangerTextColor%22%3A%22%23EC93AF%22%2C%22colorPrimaryButtonBackground%22%3A%22%23077EDF%22%2C%22colorSecondaryButtonBackground%22%3A%22%233D4042%22%2C%22colorSecondaryButtonBorder%22%3A%22%2389969F%22%2C%22borderRadius%22%3A%2210px%22%2C%22borderRadiusBadge%22%3A%225px%22%2C%22borderRadiusForm%22%3A%223px%22%2C%22spacingScale%22%3A%228px%22%2C%22colorFormHighlightBorder%22%3A%22%23363636%22%2C%22colorOffsetBackground%22%3A%22%23171717%22%2C%22borderRadiusButton%22%3A%2220px%22%7D%7D)